testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/ExtractJsonResponseAction.scala

```diff
@@ -137,4 +147,45 @@ object ExtractJsonResponseAction extends ResponseActions {
     ContentValidator.validateIntegerString(listIndex, s"ExtractJson.$StringFromList.listIndex")
   }
 
+  /**
+   * This method validates the parameters of the StringFromJsonPath type of response action.
+   *
+   * @param assertion The ResponseAction instance containing the response action details.
+   */
+  private def validateStringFromJsonPath(assertion: ResponseAction): Unit = {
+    val cacheKey = assertion.params.getOrElse("cacheKey", throw new IllegalArgumentException(s"Missing required 'cacheKey' parameter for extract $StringFromJsonPath logic"))
+    val jsonPath = assertion.params.getOrElse("jsonPath", throw new IllegalArgumentException(s"Missing required 'jsonPath' parameter for extract $StringFromJsonPath logic"))
+    val cacheLevel = assertion.params.getOrElse("cacheLevel", throw new IllegalArgumentException(s"Missing required 'cacheLevel' parameter for extract $StringFromJsonPath logic"))
```

Comment:
Since I see the literals `cacheKey`, `jsonPath`, and `cacheLevel` being reused quite extensively, wouldn't it make sense to define these in a class/object and reference such a defined value instead?

Reply:
This is connected with Issue https://github.com/AbsaOSS/ScAPI/issues/30, where I expect this will happen.